Dockerfile

```diff
@@ -1,7 +1,38 @@
-FROM python:3.9-bullseye
+FROM ubuntu:22.04
 
-# Remove after direct git reference to Prophet is removed.
-RUN apt-get update && apt-get -y install git
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+# install dependencies
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get remove -y python3.10 && \
+    apt-get install -y --no-install-recommends software-properties-common gnupg2 && \
+    add-apt-repository -y ppa:deadsnakes/ppa && \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends \
+    build-essential curl \
+    libpq-dev \
+    libsasl2-dev \
+    libssl-dev libffi-dev \
+    python3.9 python3.9-dev python3.9-venv libpython3.9-dev libpython3.9 \
+    python3.9-distutils \
+    unixodbc-dev git && \
+    apt-get clean -qq -y && \
+    apt-get autoclean -qq -y && \
+    apt-get autoremove -qq -y && \
+    rm -rf /var/lib/apt/lists/*
+RUN ln -s /usr/bin/python3.9 /usr/bin/python && \
```

Comment:
Why is this needed? can't we just use `past install python3-pip`

Reply:
There is no python3-pip in the deadsnakes PPA